packages/eslint-plugin/migrations.json

```
@@ -11,6 +11,12 @@
       "version": "17.2.6-beta.1",
       "description": "Rename workspace rules from @nx/workspace/name to @nx/workspace-name",
       "implementation": "./src/migrations/update-17-2-6-rename-workspace-rules/rename-workspace-rules"
+    },
+    "update-19-1-0-rename-no-extra-semi": {
+      "cli": "nx",
+      "version": "19.1.0-beta.4",
```

19.1.0-beta.5 is the latest beta version released. It's not a huge deal because folks would normally only migrate to stable versions, but repos that might have migrated to it wouldn't get this migration.
```suggestion
      "version": "19.1.0-beta.6",
```

---

Thanks, @leosvelperez, pushed as part of path fix